<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>oc_reel_linking_b</class>
 <widget class="QMainWindow" name="oc_reel_linking_b">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>701</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Reel Allocation</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/images/Icon_105.ico</normaloff>:/images/Icon_105.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Production Date:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="dtpProductionDate">
        <property name="displayFormat">
         <string>dd MMM yyyy</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Sample Calculation</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout"/>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Section:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txtSection"/>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Reel No.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txtReelNo"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>OCs</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="CustomQueryView" name="lstAllOCs"/>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Selected OCs</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="CustomQueryView" name="lstSelectedOCs"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomQueryView</class>
   <extends>QTreeView</extends>
   <header>customqueryview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
